infrastructure/template.yaml

```diff
@@ -1340,7 +1346,10 @@ Resources:
           AUDIT_EVENT_BUS: !Ref CheckHmrcEventBus
           AUDIT_SOURCE: !FindInMap [EnvironmentConfiguration, !Ref Environment, DOMAINNAME]
           AUDIT_ISSUER: !Sub "{{resolve:ssm:/${CommonStackName}/verifiable-credential/issuer}}"
+          MAX_JWT_TTL: !Sub "{{resolve:ssm:${MaxJwtTtlParameter}}}"
+          JWT_TTL_UNIT: !Sub "{{resolve:ssm:${JwtTtlUnitParameter}}}"
```

Comment:
Does this really need to be an env var coming from SSM? It feels to me like it could just be a constant?

Reply:
It depends on   # Only numeric values should be assigned here

```
  MaxJwtTtl:
    Environment:
      dev: 2
      localdev: 2
      build: 2
      staging: 6
      integration: 6
      production: 6

  # Permitted values: SECONDS,MINUTES,HOURS,DAYS,MONTHS,YEARS
  JwtTtlUnit:
    Environment:
      dev: HOURS
      localdev: HOURS
      build: HOURS
      staging: MONTHS
      integration: MONTHS
      production: MONTHS
```

Reply:
Perhaps I could just use the mapping directly in the Environment

Reply:
Why is it different in different envs though? 👀